notifications.bs

```diff
@@ -70,6 +70,11 @@ that no sounds or vibrations should be made.
 indicates that the screen of the device should not be enabled.
 
 <p>A <a lt="concept notification">notification</a> has an associated
+<dfn>require interaction preference flag</dfn> which is initially unset. When
+set, hints that the notification should remain readily available until the user
```

Comment:
s/hints/indicates/? The rest of the sentences uses "... should remain ..." anyway.